vassal-doc/src/main/readme-referencemanual/ReferenceManual/SetGlobalProperty.adoc

```diff
@@ -24,29 +24,30 @@ Look for the Global property first in the Zone the piece is currently on. If not
 _Named Zone_:::
 Look for the Global Property in the Named Zone on the Map the piece is currently on. If not found, look for a Map level Global Property on the current Map, then look for a module level Global Property.
 _Named Map_:::
-Look for the a Map level Global property on the Named Map. The piece does not have to be on this map. If not found, then look for a module level Global Property.
+Look for a Map level Global property on the Named Map. The piece does not have to be on this map. If not found, then look for a module level Global Property.
 
 *Name of Zone containing property:*::
 An <<Expression.adoc#top,Expression>> defining the name of the Zone to look in for the Global Property.
 
 *Name of Map containing property:*::
 An <<Expression.adoc#top,Expression>> defining the name of the Map to look in for the Global Property.
 
+*Is numeric:*:: If true, then changes to the value of the property by this trait will be restricted to integer values.
 *Is numeric:*:: If true, then changes to the value of the property by this trait will be restricted to integer values.
 
 *Minimum value:*::  Numeric values will be restricted to no less than this number when changed by this trait.
 
 *Maximum value:*::  Numeric values will be restricted to no more than this number when changed by this trait.
 
-*Wrap Around:*::  If true, then when incrementing this numeric property, values will wrap around from the maximum to the minimum and vice versa.
+*Wrap:*::  If true, then when incrementing this numeric property, values will wrap around from the maximum to the minimum and vice versa.
 
-*Key Commands:*::  Adds any number commands for this piece, each of which will set change the value of the designated property in the specified way.
+*Key commands:*::  Adds any number commands for this piece, each of which will change the value of the designated property in the specified way.
```

Comment:
Missing "of" between "number" and "commands" (sorry, not sure why I missed it the first time!)